planning/behavior_velocity_planner/src/scene_module/run_out/scene.cpp

```diff
@@ -140,9 +140,7 @@ Polygons2d RunOutModule::createDetectionAreaPolygon(const PathWithLaneId & smoot
   auto stop_dist = run_out_utils::calcDecelDistWithJerkAndAccConstraints(
     initial_vel, target_vel, initial_acc, planning_dec, jerk_acc, jerk_dec);
 
-  if (!stop_dist) {
-    *stop_dist = 0;
-  }
+  const double stop_distance = !stop_dist ? 0 : *stop_dist;
```

Comment:
@TomohitoAndo 
what is the motivation to use optional?
it' seems to be better to return 0 if stop_dist is invalid in your suggestion.

Reply:
It is because there are two similar variables `stop_dist` and `stop_distance`, and it looks redundant.
But both ways do the same thing, I think it's okay. :+1:

Reply:
> it looks redundant
agree , OK I will accept your change